display_applications/igv/gff.xml

```diff
@@ -0,0 +1,110 @@
+                $site_organisms[ $site_dbkeys.index( $gff_file.dbkey ) ]
+            #else:
+                $gff_file.dbkey
+            #end if
+        </param>
+        
+        <param type="template" name="jnlp" url="galaxy_${DATASET_HASH}.jnlp" viewable="True" mimetype="application/x-java-jnlp-file">&lt;?xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;?&gt;
+&lt;jnlp
+  spec=&quot;1.0+&quot;
+  codebase=&quot;${site_link}&quot;&gt;
+  &lt;information&gt;
+    &lt;title&gt;IGV 1.5&lt;/title&gt;
+    &lt;vendor&gt;The Broad Institute&lt;/vendor&gt;
+    &lt;homepage href=&quot;http://www.broadinstitute.org/igv&quot;/&gt;
+    &lt;description&gt;IGV Software&lt;/description&gt;
+    &lt;description kind=&quot;short&quot;&gt;IGV&lt;/description&gt;
+  &lt;/information&gt;
+  &lt;security&gt;
+      &lt;all-permissions/&gt;
+  &lt;/security&gt;
+  &lt;resources&gt;
+  
+&lt;j2se version=&quot;1.5+&quot; initial-heap-size=&quot;256m&quot; max-heap-size=&quot;1100m&quot;/&gt;    
+	&lt;jar href=&quot;igv.jar&quot; download=&quot;eager&quot; main=&quot;true&quot;/&gt;
+    &lt;jar href=&quot;batik-codec.jar&quot; download=&quot;eager&quot;/&gt; 
+    &lt;property name=&quot;apple.laf.useScreenMenuBar&quot; value=&quot;true&quot;/&gt;
+    &lt;property name=&quot;com.apple.mrj.application.growbox.intrudes&quot; value=&quot;false&quot;/&gt;
+    &lt;property name=&quot;com.apple.mrj.application.live-resize&quot; value=&quot;true&quot;/&gt;
+    &lt;property name=&quot;com.apple.macos.smallTabs&quot; value=&quot;true&quot;/&gt;
+  &lt;/resources&gt;
+	
+    &lt;resources os=&quot;Mac&quot; arch=&quot;i386&quot;&gt;
+        &lt;property name=&quot;apple.awt.graphics.UseQuartz&quot; value=&quot;false&quot;/&gt;
+        &lt;nativelib href=&quot;hdfnative-macintel.jar&quot;/&gt; 
+    &lt;/resources&gt;
+
+    &lt;resources os=&quot;Mac&quot; arch=&quot;ppc&quot;&gt;
+        &lt;property name=&quot;apple.awt.graphics.UseQuartz&quot; value=&quot;false&quot;/&gt;
+        &lt;nativelib href=&quot;hdfnative-macppc.jar&quot;/&gt; 
+    &lt;/resources&gt;
+    
+    &lt;resources os=&quot;Mac&quot; arch=&quot;PowerPC&quot;&gt;
+        &lt;property name=&quot;apple.awt.graphics.UseQuartz&quot; value=&quot;false&quot;/&gt;
+        &lt;nativelib href=&quot;hdfnative-macppc.jar&quot;/&gt; 
+    &lt;/resources&gt;
+
+    &lt;resources os=&quot;Windows&quot;&gt;
+        &lt;property name=&quot;sun.java2d.noddraw&quot; value=&quot;true&quot;/&gt;
+        &lt;nativelib href=&quot;hdfnative-win.jar&quot;/&gt; 
+    &lt;/resources&gt;
+
+    &lt;resources os=&quot;Linux&quot;&gt;
+        &lt;nativelib href=&quot;hdfnative-linux64.jar&quot;/&gt; 
+    &lt;/resources&gt;
+
+  &lt;application-desc main-class=&quot;org.broad.igv.ui.IGVMainFrame&quot;&gt;
+     &lt;argument&gt;-g&lt;/argument&gt;
+     &lt;argument&gt;${site_organism}&lt;/argument&gt;
+     &lt;argument&gt;${gff_file.url}&lt;/argument&gt;
+  &lt;/application-desc&gt;
+&lt;/jnlp&gt;

```

Comment:
That is probably an option, but then I'd have to do more than copy&paste with some modifications from the other known-working IGV displays.